fuzzers/baby/baby_fuzzer_intel_pt/src/main.rs

```diff
@@ -0,0 +1,153 @@
+use std::{hint::black_box, num::NonZero, path::PathBuf, process, time::Duration};
```

Comment:
This should go into `binary_only,` right?

Reply:
Actually I wanted to ask you this:
ATM I have 2 example fuzzers, this one `baby_fuzzer_intel_pt`, that is basically the inprocess babyfuzzer with PT instead of instrumentation (just to be as simple as possible) and the `command_executor_intel_pt` which is essentially a linux binary only.
Where do you think is best to put them?

Reply:
binary_only should go into binary_only I think.
The simple one is up to you. Although people that are looking for binary-only fuzzing won't hit the baby folder, so I would still put it in binary_only, _personally_.

Reply:
same opinion here

Reply:
Ok thanks, I moved both to `binary_only`